rebar.config

```diff
@@ -61,22 +62,33 @@
 
 %% == Dependencies ==
 
-{deps, [ {xref_runner, "1.1.1"}
-       , {elvis_core, "0.4.1"}
+{deps, [ {xref_runner, "1.2.0"}
+       , {elvis_core, "1.1.2"}
        ]}.
 
+{project_plugins, [
+    rebar3_hank,
+    rebar3_hex,
+    rebar3_lint
+]}.
+
 %% == Dialyzer ==
 
-{dialyzer, [ {warnings, [ no_return
-                        , unmatched_returns
-                        , error_handling
+{dialyzer, [ {warnings, [ error_handling,
```

Comment:
Simplified to reduce error-proneness. And also to avoid `rebar3 dialyzer || rebar3 dialyzer`.